components/ButtonCounter.styles.js

```diff
@@ -18,10 +18,10 @@ export const styles = StyleSheet.create({
   },
   highlightUpStyle: {
     color: COLORS.primary,
-    fontWeight: FONTWEIGHT.bold
+    fontFamily: FONTS.familySemiBold
   },
   highlightDownStyle: {
     color: COLORS.danger,
-    fontWeight: FONTWEIGHT.bold
+    fontFamily: FONTS.familySemiBold
```

Comment:
using fontfamily here instead of bold sans text styling as it matches the design